sky/provision/kubernetes/instance.py

```diff
@@ -282,7 +291,7 @@ def _set_env_vars_in_pods(namespace: str, new_pods: List):
 
     for new_pod in new_pods:
         _run_command_on_pods(new_pod.metadata.name, namespace,
-                             set_k8s_env_var_cmd)
+                             set_k8s_env_var_cmd, False)
```

Comment:
If we make `stream_logs` arg optional, it's good to explicitly specify the arg name for readability, especially for bool args:

```
_run_command_on_pods(new_pod.metadata.name, namespace, set_k8s_env_var_cmd, stream_logs=False)
```